packages/eslint-plugin/src/rules/class-literals-style.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import * as util from '../util';
+
+type Options = [('fields' | 'getters')?];
+type MessageIds = 'preferFieldStyle' | 'preferGetterStyle';
+
+interface NodeWithModifiers {
+  accessibility?: TSESTree.Accessibility;
+  static: boolean;
+}
+
+const printNodeModifiers = (
+  node: NodeWithModifiers,
+  final: 'readonly' | 'get',
+): string =>
+  `${node.accessibility ?? ''}${
+    node.static ? ' static' : ''
+  } ${final} `.trimLeft();
+
+const isSupportedLiteral = (
+  node: TSESTree.Node,
+): node is TSESTree.LiteralExpression =>
+  node.type === AST_NODE_TYPES.Literal ||
+  node.type === AST_NODE_TYPES.BigIntLiteral ||
+  (node.type === AST_NODE_TYPES.TemplateLiteral && node.quasis.length === 1);
```

Comment:
Very good point! I've added support, with the same bailout on if they've got any expressions since we'd have to scope-check them which is probably overkill (unless I've missed the magic method that checks exactly that?).

I've also not had the fixer bother trying to make them look nicer by having them multi-line, as that feels like more overkill as we'd have to try to calculate indentation.

Let me know what you think :)